### DISCUSSION TITLE
pyMOR Community Meeting 2023-11-14

### DISCUSSION THREAD

**Discussion by HenKlei** [pymor/pymor#2218] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, November 14, at 15:00 CET and 9:00 EST, online at:

Link: http://bbb.mpi-magdeburg.mpg.de/meeting/pymor
Password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [ ] https://github.com/pymor/pymor/pull/2162 (@pmli)
- [ ] 2023.1.1 release (@pmli)
- [ ] QR interface (@sdrave)
- [ ] Attributions (@sdrave)
- [ ] https://github.com/pymor/pymor/pull/2070 (@pmli)

Moderator: @HenKlei

**Comment by lbalicki** [comment]
Notes:
- yamlfix hook
  - it formated conda hooks -> this is now turned off
  - some criticism:
    - lists are manipulated kind of arbitrarily
    - accepting all or just one kind of list should probably be accepted
    - but indentations and quotes looks good
  - 2023.1.1 release
    - binder is not working
    - problem: forgot to merge backported PR which pulls the correct github
    - backport binder setup and do another point release to add the fixed binder setup
  - QR interface
    - Question: Should qr return full n x m matrix for Q even if input is not full rank?
    - Different opinions:
      - If qr input is rank deficient Q should still have dimension of the input even though the input is rank deficient
      - If qr input is rank deficient there should be an error
    - Question: Should there be rrqr or qr with various input parameters -> Kind of depends on decision on behavior
    - Will have to think about it more and postpone decision on this matter
    - @pmli will add some comments
    - Cholesky QR cannot deal with zero columns -> Add some more tests
- Attributions 
  - For next PR contributions are not listed on authors.md
  - Should we stick with policy that contributors should add themselves or should we take care of that while writing the release notes
  - Let release manager take care of it
  - Need to update Contributing.md and Release process file
  - Use information from contributers github page (if they show name and email use that otherwise use github account name)
-  More linting and formatting
  - need to check how the changes in the randomization affect testing and randomization in pyMOR
  - there are a lot of commits adding various linting rules
  - should go through it commit by commit

Next Meeting: November 28, at 15:00 CET and 9:00 EST, Moderator @sdrave
